### ISSUE TITLE
Reading and saving a project with Xcodeproj breaks emoji characters in the project

### ISSUE THREAD

**Issue by kylef** [CocoaPods/Xcodeproj#196]
Before:

![screen shot 2014-09-30 at 17 02 42](https://cloud.githubusercontent.com/assets/44164/4461591/a4ff711e-48bb-11e4-8927-3abedfb38fd1.png)

Xcodeproj:

```ruby
require 'xcodeproj'
project = Xcodeproj::Project.open('Palaver.xcodeproj')
project.save
```

After:

![screen shot 2014-09-30 at 17 03 45](https://cloud.githubusercontent.com/assets/44164/4461596/ad2682e2-48bb-11e4-9f8c-5d36b7598595.png)

**Comment by neonichu** [comment]
This has been fixed by #203 :boom:

**Comment by 0xced** [comment]
For the record, [rdar://18512876][1] is fixed in OS X 10.11.

  [1]: http://openradar.appspot.com/18512876

**Comment by alloy** [comment]
Nice
